src/Binding/JSSystem.cpp

```diff
@@ -119,7 +111,7 @@ JSFunctionSpec *JSSystem::ListMethods()
 void JSSystem::RegisterObject(JSContext *cx)
 {
     JSSystem::ExposeClass<0>(cx, "System");
-    JSSystem::CreateUniqueInstance(cx, new JSSystem(), "System");
+    JSSystem::CreateUniqueInstance(cx, new JSSystem(), "NidiumSystem");
```

Comment:
No it was on ExposeClass not CreateUniqueInstance.
Also the new doc is wrong then.